exercises/strain/strain_test.go

```diff
@@ -90,8 +90,8 @@ func TestDiscardInts(t *testing.T) {
 		}
 		switch res := cp.Discard(test.pred); {
 		case !reflect.DeepEqual(cp, test.list):
-			t.Fatalf("Ints%v.Discard() should not modify its reciever.  "+
-				"Found %v, reciever should stay %v",
+			t.Fatalf("Ints%v.Discard() should not modify its receiver.  "+
```

Comment:
I was too focused on the strings, I totally overlooked this. Thanks @ferhatelmas